UltraStar Play/Assets/Common/Model/Song/Builder/SongMetaBuilder.cs

```diff
@@ -112,12 +112,23 @@ public static SongMeta ParseFile(string path, Encoding enc = null)
                 }
             }
 
+			//Read the song file body
+            string songBodyText = "";
+            string bodyLine;
+            while ((bodyLine = reader.ReadLine()) != null)
+                songBodyText += bodyLine; //Ignorning the newlines for the hash
+
+            //Hash the song file body
+            var songHash = Hashing.MD5(Encoding.UTF8.GetBytes(songBodyText));
+            Debug.Log("Hash for " + requiredFields["title"] + " is " + songHash);
```

Comment:
For the code style, we decided not to use type inference. So please always use the explicit type instead of var keyword.